draft-ietf-quic-manageability.md

```diff
@@ -612,24 +620,29 @@ version specific initial salt, as described in {{QUIC-TLS}}. The length of
 the connection ID is indicated in the 6th byte of the header followed by the
 connection ID itself.
 
-To determine the end of the header and find the start of the payload further
-the packet number length, the source connection ID length, as well as the token
+To determine the end of the header and find the start of the payload,
+the packet number length, the source connection ID length, and the token
 length need to be extracted. The packet number length is defined by the seventh
 and eight bits of the header as described in section 17.2. of
-{{QUIC-TRANSPORT}}. The source connection ID length is specified in the byte
-after the destination connection ID. And the token length, which follows the
-source connection ID, is a variable length integer as specified in section 16
-of {{QUIC-TRANSPORT}}.
-
-Finally after decryption, the Initial Client packet can be parsed to detect the
-CRYPTO frame that contains the TLS Client Hello, which then can be respectively
-parsed similar as for all other TLS connections. The Initial client packet may
-contain other frames, so the first byte of each frame need to be checked to
-identify the frame type and the skip over the frame. Note that the length of the
-frames is dependent on the frame type. Usually for QUIC version 1, the packet is
+{{QUIC-TRANSPORT}}, but is obfuscated as described in {{QUIC-TLS}}. The source
+connection ID length is specified in the byte after the destination connection
+ID. And the token length, which follows the source connection ID, is a variable
+length integer as specified in section 16 of {{QUIC-TRANSPORT}}.
+
+After decryption, the Initial Client packet can be parsed to detect the
+CRYPTO frame that contains the TLS Client Hello, which then can be parsed
+similarly to TLS over TCP connections. The Initial client packet may
+contain other frames, so the first bytes of each frame need to be checked to
+identify the frame type, and if needed skip over it. Note that the length of the
+frames is dependent on the frame type. In QUIC version 1, the packet is
 expected to only carry the CRYPTO frame and optionally padding frames. However,
-padding which is one byte of zeros, may also occur before or after the CRYPTO
-frame.
+PADDING frames, which are each one byte of zeros, may also occur before or after
+the CRYPTO frame.
+
+Note that client Initial packets that do not contain the Client Hello message
```

Comment:
I read this at first and thought "wat?", but I think that you just have an extra negation here:

```suggestion
Note that client Initial packets that contain the Client Hello message
```

Reply:
What I'm trying to refer to here is the Initial ACK. Observers should expect that a stateless DPI will like fail to decrypt ~50% of client Initial packets. I think this proposed change inverts the meaning?

Reply:
ACK.  I just misread this.  The double-negation caught me out.  I think that you might want to try to work on this.  Maybe "client Initial packets other than those in the very first flight sent by the client"

It's got nothing to do with the ClientHello, it's whether the client has seen packets from the server yet.